app/client/src/components/designSystems/appsmith/FilePickerComponent.tsx

```diff
@@ -17,6 +17,9 @@ class FilePickerComponent extends React.Component<
   }
 
   openModal = () => {
+    if (this.props.isDisabled) {
+      return;
+    }
     this.props.uppy.getPlugin("Dashboard").openModal();
```

Comment:
```suggestion
    if (!this.props.isDisabled) {
     this.props.uppy.getPlugin("Dashboard").openModal();
    }
    
```
not much of a comment. this just looks cleaner to me.

Reply:
Thanks @marks0351